mkdocs.yml

```diff
@@ -2,7 +2,11 @@ site_name: Zanata Server
 docs_dir: docs
 repo_name: GitHub
 repo_url: http://www.github.com/zanata/zanata-server
+site_favicon: images/favicon.ico
```

Comment:
Should we use the one from assets? http://assets-zanata.rhcloud.com/master/favicon.ico